docs/application/dotnet/guides/nui/resources.md

```diff
@@ -0,0 +1,87 @@
+# Resources
+
+NUI provides several ways to handle resource images.
+This tutorial describes the use of Image resources in NUI.
+
+## Overview
+
+Resources in NUI can apply Images or 3D Models.
+
+The common method to access these resources is through Visuals.
+Controls use these visuals to display what is required.
+
+See [ImageView tutorial](imageview.md).
+
+## Loading images
+
+Images are defined by an url which can be local or remote (internet).
```

Comment:
Images are defined by an url which can be local or remote (internet). 
change
url to "URL"